invenio_vocabularies/contrib/funders/mappings/v6/funders/funder-v1.0.0.json

```diff
@@ -0,0 +1,55 @@
+{
+  "mappings": {
+    "_doc": {
+      "dynamic": "strict",
+      "properties": {
+        "$schema": {
+          "type": "keyword",
+          "index": "false"
+        },
+        "created": {
+          "type": "date"
+        },
+        "updated": {
+          "type": "date"
+        },
+        "uuid": {
```

Comment:
(this is more of a exploratory comment)

question: what does this field get as value? and why is it needed? If I recall correctly the `ObjectId` in Elasticsearch is has the same value than the record UUID (which gets linked to the PID value via PIDStore). When denormalizing a vocabulary record into a record we add a `@v` value in order to be able to version changes and update correctly. Is this UUID value needed in order to not query again when performing the denormalization?

(We can debug this together, I am just noting this here to not forget)

Reply:
https://github.com/inveniosoftware/invenio-vocabularies/issues/161